example/App.js

```diff
@@ -24,6 +24,25 @@ export default class App extends Component {
     Alert.alert(`Package: ${packageSearch}`, `${isInstalled ? 'Installed' : 'Not Installed'}`);
   };
 
+  constructor(props: Props) {
```

Comment:
You initialise this message on the state = {} on top of the component. Since there is already a state declaration upper

Reply:
Removed constructor